Vlv2TbltDevicePkg/CustomShellPkg/Application/Shell/Shell.c

```diff
@@ -314,6 +315,219 @@ InternalEfiShellStartCtrlSMonitor(
   return (Status);
 }
 
+/**
+  internal worker function to load and run an image via memory-mapped device path.
```

Comment:
@krystian-hebel please be consistent: either you use fits bit letter either not.